heatpump.yaml

```diff
@@ -4547,3 +4525,86 @@ text_sensor:
         - 141 -> L7
         - 142 -> L8
         - 143 -> L9
+  # Register: 101
+  - platform: modbus_controller
+    modbus_controller_id: "${devicename}"
+    name: "Operating Mode"
+    id: "${devicename}_operating_mode"
+    icon: mdi:state-machine
+    register_type: holding
+    address: 0x65
+    response_size: 2
+    lambda: |-
+        int idx = item->offset;
+        uint16_t rawdata = (uint16_t(data[idx]) << 8) + uint16_t(data[idx + 1]);
+        std::string output = "Unknown: ";
+        output += to_string(rawdata);
+        if (rawdata == 0) output = "OFF";
+        else if (rawdata == 2) output = "Cooling";
+        else if (rawdata == 3) output = "CO Heating";
+        else if (rawdata == 5) output = "DHW Heating";
```

Comment:
I do not see this one, when DHW is heating, the output is still 3. Where/when did you see the output 5?